Model/Config/Frontend/EnableProductProtection.php

```diff
@@ -101,4 +111,14 @@ private function checkIfV1PPEnabledInAnyStores()
 
         return in_array(1, $values);
     }
+
+    /**
+     * Check if the new Product Protection is enabled
+     *
+     * @return mixed
+     */
+    private function checkIfV2ExtendEnabled()
+    {
```

Comment:
this isn't really checking if product protection is enabled right? It's checking whether the module is enabled. Is it the wrong flag or is the name weird?

Reply:
Yes, it's checking if the module is enabled.  PP can't be enabled if the module is disabled.